lib/livebook/file_system/s3.ex

```diff
@@ -398,17 +398,21 @@ defimpl Livebook.FileSystem, for: Livebook.FileSystem.S3 do
     host |> String.split(".") |> Enum.reverse() |> Enum.at(2, "auto")
   end
 
-  defp encode({:ok, status, headers, body}) do
+  defp decode({:ok, status, headers, body}) do
     case HTTP.fetch_content_type(headers) do
       {:ok, content_type} when content_type in ["text/xml", "application/xml"] ->
         {:ok, status, headers, XML.decode!(body)}
 
+      # Google Cloud Storage XML API returns this type of response.
+      {:ok, "text/html"} when body == "" ->
+        {:ok, status, headers, body}
+
       :error ->
```

Comment:
We should just change to `_ ->`, so if there is any other content type we keep the body as is and this should handle the `text/html` case :)

Reply:
That's for sure! https://github.com/livebook-dev/livebook/pull/956/commits/5dd41c9fe30180707031c93704de249970723197